Logger.Tests/EntityTests.cs

```diff
@@ -0,0 +1,73 @@
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace Logger.Tests;
+
+[TestClass]
+public class EntityTests
+{
+    [TestMethod]
+    public void Students_With_Same_Properties_Are_Equal()
+    {
+        Student student1 = new Student
+        {
+            Id = Guid.NewGuid(),
+            FullName = new FullName("Spongebob", "SquarePants"),
+            StudentYear = StudentLevels.Freshman
+        };
+
+        Student student2 = new Student
+        {
+            Id = student1.Id,
+            FullName = student1.FullName,
+            StudentYear = student1.StudentYear
+        };
+
+        Assert.AreEqual(student1, student2);
+    }
```

Comment:
```suggestion
    {
       // Arrange
        Student student1 = new Student
        {
            Id = Guid.NewGuid(),
            FullName = new FullName("Spongebob", "SquarePants"),
            StudentYear = StudentLevels.Freshman
        };

        Student student2 = new Student
        {
            Id = student1.Id,
            FullName = student1.FullName,
            StudentYear = student1.StudentYear
        };
        
        // Act & Assert
        Assert.AreEqual(student1, student2);
    }
```
Consider using the // Arrange // Act // Assert formatting